### PR TITLE
Make END button redirect to courses page

### PR DESCRIPTION
**Pull request by Tawakalt**
#### What does this PR do?
- It makes the `END` button at the end of a course redirect to the courses page.

#### Description of task completed
- Instead of redirecting to the beginning of a course at the click of the `END` button, it now redirects to the courses page.

#### Link to test
- https://cfafrica-mooc-dev.appspot.com/test/unit?unit=4&lesson=13

### CODE REVIEW

File: coursebuilder/views/unit.html [hunk 1]
```diff
@@ -130,7 +130,7 @@
                 {% if next_button_url %}
                   <a href="{{ next_button_url }}" class="btn right">{{ gettext('Next') }}</a>
                 {% else %}
-                  <a href="course" class="btn right"> {{ gettext('End') }} </a>
+                  <a href="/#!explorer" class="btn right"> {{ gettext('End') }} </a>
```
Comment (chegejames) [inline coursebuilder/views/unit.html:133]: @Tawakalt Have you tested this?

Comment (chegejames) [reply coursebuilder/views/unit.html:133]: @Tawakalt this is only supposed to change for the last unit of a course.

Comment (Tawakalt) [reply coursebuilder/views/unit.html:133]: @chegejames Yes I have. The END button appears only in the last unit of a course.

Comment (chegejames) [reply coursebuilder/views/unit.html:133]: @Tawakalt it also appears in the last lesson of a unit e.g https://cfafrica-mooc-dev.appspot.com/test/unit?unit=1&lesson=6

Comment (Tawakalt) [reply coursebuilder/views/unit.html:133]: @chegejames Didn't see that. Will fix it.

**Pull request closed without merging**